packages/fluentui/react-northstar/src/components/Pill/PillContent.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import { Accessibility } from '@fluentui/accessibility';
+import {
+  ComponentWithAs,
+  getElementType,
+  useUnhandledProps,
+  useAccessibility,
+  useFluentContext,
+  useStyles,
+  useTelemetry,
+} from '@fluentui/react-bindings';
+import {
+  childrenExist,
+  UIComponentProps,
+  ChildrenComponentProps,
+  ContentComponentProps,
+  commonPropTypes,
+  rtlTextContainer,
+} from '../../utils';
+
+import { FluentComponentStaticProps } from '../../types';
+import { PillProps } from './Pill';
+
+export interface PillContentProps extends UIComponentProps, ChildrenComponentProps, ContentComponentProps {
+  /**
+   * Accessibility behavior if overridden by the user.
+   */
+  accessibility?: Accessibility<never>;
+
+  /**
+   * A Pill can be sized.
+   */
+  size?: PillProps['size'];
```

Comment:
This seems strange... so I can set a different size value for `Pill` and `PillContent` at the same time ?

Reply:
any way to do some typescript fu so that this is prop is not in a public interface ?

Reply:
I think this is the standard we are following in other components as well, is that right @layershifter ?

Reply:
Let's not couple typings there, options:
- add `PillSize` type
- duplicate type definition

```suggestion
  size?: Extract<SizeValue, 'smaller' | 'small' | 'medium'>;
```